packages/widget/src/pages/SwapExecutionPage/SwapExecutionPage.tsx

```diff
@@ -175,6 +179,10 @@ export const SwapExecutionPage = () => {
         return (
           <MainButton label="Confirm swap" icon={ICONS.rightArrow} loading />
         );
+      case SwapExecutionState.approving:
+        return (
+          <MainButton label="Awaiting approval" icon={ICONS.rightArrow} loading />
```

Comment:
"approval" seems too generic. Let's change this to "Awaiting allowance approval"